pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/VenueImageEdit/__specs__/VenueImageEdit.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import { fireEvent, render } from '@testing-library/react'
+import React from 'react'
+
+import { VenueImageEdit } from '../VenueImageEdit'
+
+const mockCloseModal = jest.fn()
+const defaultProps = {
+  image: new File([], 'toto.png', {
+    type: 'image/png',
+  }),
+  onSetImage: jest.fn(),
+  closeModal: mockCloseModal,
+}
+
+describe('venue image edit', () => {
+  it('closes the modal on cancel button click', () => {
+    const { getByText } = render(<VenueImageEdit {...defaultProps} />)
+    fireEvent.click(getByText('Annuler'))
+    // We don't want to use toHaveBeenCalledWith, as it's been called with a huge mouse click event
+    // eslint-disable-next-line jest/prefer-called-with
+    expect(mockCloseModal).toHaveBeenCalled()
```

Comment:
```suggestion
    expect(mockCloseModal).toHaveBeenCalledTimes(1)
```
Tu peux aussi utiliser `toHaveBeenCalledTimes(1)`, ça évite d'ajouter la ligne de eslint-disable